tables/mdm/mdm.go

```diff
@@ -65,22 +77,17 @@ func MDMInfoColumns() []table.ColumnDefinition {
 }
 
 func MDMInfoGenerate(ctx context.Context, queryContext table.QueryContext) ([]map[string]string, error) {
-	profiles, err := getMDMProfile()
-	if err != nil {
-		return nil, err
-	}
+	profiles, _ := getMDMProfile()
```

Comment:
Why did you stop handling the err and instead throw it out? Please leave a comment to justify.